pkg/sources/gitlab/gitlab.go

```diff
@@ -577,6 +582,7 @@ func (s *Source) getAllProjectRepos(
 			ListOptions:      listOpts,
 			OrderBy:          gitlab.Ptr(orderBy),
 			IncludeSubGroups: gitlab.Ptr(true),
+			WithShared:       gitlab.Ptr(s.enumerateSharedProjects),
```

Comment:
The [docs](https://docs.gitlab.com/ee/api/groups.html#list-a-groups-projects) indicate `with_shared` defaults to `true`. Is this config option here to disable `with_shared` for debug testing?

Reply:
yes exactly